cypress/integration/data-import/mrc-import-with-matching-001.spec.js

```diff
@@ -0,0 +1,144 @@
+    const nameForCSVFile = `autoTestFile${getRandomPostfix()}.csv`;
+
+    // unique name for profiles
+    const matchProfileName = `autoTestMatchProf.${getRandomPostfix()}`;
+    const mappingProfileName = `autoTestMappingProf.${getRandomPostfix()}`;
+    const actionProfileName = `autoTestActionProf.${getRandomPostfix()}`;
+    const jobProfileName = `autoTestJobProf.${getRandomPostfix()}`;
+
+    // upload a marc file for export
+    dataImport.uploadFile(nameForMarcFile);
+    jobProfiles.searchJobProfileForImport('Default - Create instance and SRS MARC Bib');
+    jobProfiles.runImportFile(nameForMarcFile);
+    logs.openJobProfile(nameForMarcFile);
+    logs.checkIsInstanceCreated();
+
+    // get Instance HRID through API
+    SearchInventory
+      .getInstanceHRID()
+      .then(id => {
+        // download .csv file
+        SearchInventory.gotoInventory();
+        SearchInventory.searchInstanceByHRID(id);
+        inventorySearch.saveUUIDs();
+        ExportMarcFile.downloadCSVFile(nameForCSVFile, 'SearchInstanceUUIDs*');
+        FileManager.deleteFolder(Cypress.config('downloadsFolder'));
+        cy.visit(TopMenu.dataExport);
+
+        // download exported marc file
+        exportFile.uploadFile(nameForCSVFile);
+        exportFile.exportWithDefaultInstancesJobProfile(nameForCSVFile);
+        ExportMarcFile.downloadExportedMarcFile(nameForExportedMarcFile);
+        FileManager.deleteFolder(Cypress.config('downloadsFolder'));
+
+        // create Match profile
+        const matchProfile = {
+          profileName: matchProfileName,
+          incomingRecordFields: {
+            field: '001',
+          },
+          existingRecordFields: {
+            field: '001',
+          },
+          matchCriterion: 'Exactly matches',
+        };
+
+        SettingsDataImport.goToMatchProfile();
+
+        MatchProfiles.createMatchProfile(matchProfile);
+
+        // create Field mapping profile
+        const mappingProfile = {
+          name: mappingProfileName,
+          typeValue : NewFieldMappingProfile.folioRecordTypeValue.instance,
+          update: true
+        };
+
+        SettingsDataImport.goToMappingProfile();
+        FieldMappingProfiles.createMappingProfile(mappingProfile);
+
+        // create Action profile and link it to Field mapping profile
+        const actionProfile = {
+          typeValue : NewActionProfile.folioRecordTypeValue.instance,
+          name: actionProfileName,
+          action: 'Update (all record types except Orders)',
+        };
+        SettingsDataImport.goToActionProfile();
+        ActionProfiles.createActionProfile(actionProfile, mappingProfile);
+        ActionProfiles.checkActionProfilePresented(actionProfileName);
+
+        // create Job profile
+        const jobProfile = {
+          ...NewJobProfile.defaultJobProfile,
+          profileName: jobProfileName
+        };
+        SettingsDataImport.goToJobProfile();
+        jobProfiles.openNewJobProfileForm();
```

Comment:
you can try to wrap such methods to something like jobProfiles.create():
jobProfiles.openNewJobProfileForm();
        NewJobProfile.fillJobProfile(jobProfile);
        NewJobProfile.linkMatchAndActionProfiles(matchProfileName, actionProfileName);
        NewJobProfile.clickSaveAndCloseButton();
        jobProfiles.waitLoadingList();